### ISSUE TITLE
Search in Calibre produces no results

### ISSUE THREAD

**Issue by o20n3** [fallaciousreasoning/CalibreLibgenStore#8]
Can see the store in Calibre but search doesn't return any titles.

**Comment by groseler** [comment]
+1

**Comment by Protektor-Desura** [comment]
+1 even though there are results when you search on the website

**Comment by fallaciousreasoning** [comment]
I suspect this is something to do with the filtering Calibre applies to the results after they come in (but I'm not sure). I'm not really working on this anymore (I generally try and keep it working but that's about it).

I just released a fix for broken downloads and it's (possible) that this will fix it. If it does, would you mind commenting/closing this. If not, I'm afraid I'm probably not going to get to it in the near future sorry.

**Comment by Protektor-Desura** [comment]
Yes that fixed it and I am getting results now. But I get failed download now.
"Exception: No file specified to download." But I had just clicked on an EPUB version it listed.

**Comment by Tenome** [comment]
This plugin's sorta janky, until it gets worked on some more I'd just use Libgen Desktop Search instead. Much faster, will give you all the results as expected, can easily switch mirrors, etc.
